packages/frontend/src/model/welcome_overlay.css

```diff
@@ -0,0 +1,198 @@
+    transition: opacity 0.3s ease, visibility 0.3s ease;
+    z-index: 10;
+}
+
+.welcome-overlay.open {
+    opacity: 1;
+    visibility: visible;
+}
+
+.welcome-overlay-content {
+    background: white;
+    padding: 20px;
+    border-radius: 8px;
+    box-shadow: rgba(0, 0, 0, 0.1) 0px 4px 8px;
+    /* max-width: 600px; */
+    width: 90%;
+    height: 90%;
+}
+
+.nav-button.get-started {
+    border: none;
+    border-radius: 5px;
+    cursor: pointer;
+}
+
+.get-started:hover {
+    background-color: #45a19d; /* Darker shade for hover effect */
+}
+
+.welcome-content {
+    background: white;
+    border-radius: 12px;
+    box-shadow: 0 10px 25px rgba(0, 0, 0, 0.2);
+    max-width: 700px;
+    width: 700px;
+    /* height: 750px; */
+    position: relative;
+    overflow: hidden;
+    display: flex;
+    flex-direction: column;
+}
+
+.close-button {
+    position: absolute;
+    top: 15px;
+    right: 15px;
+    background: transparent;
+    border: none;
+    cursor: pointer;
+    color: #666;
+    z-index: 10;
+    transition: color 0.2s;
+    padding: 5px;
+    border-radius: 50%;
+}
+
+.close-button:hover {
+    color: #333;
+    background: rgba(0, 0, 0, 0.05);
+}
+
+.fade-in {
+    animation: fadeIn 0.4s ease-in-out;
+}
+
+@keyframes fadeIn {
+    from {
+        opacity: 0;
+        transform: translateY(10px);
+    }
+    to {
+        opacity: 1;
+        transform: translateY(0);
+    }
+}
+
+.welcome-body {
+    padding: 25px 35px;
+    min-height: 250px;
+    overflow-y: auto;
+}
+
+.welcome-body header {
+    text-align: center;
+    margin-bottom: 30px;
+}
+
+.welcome-body header h1 {
+    font-size: 28px;
+    font-weight: bold;
+    margin-bottom: 10px;
+    color: var(--main-color);
+}
+
+.welcome-body header p {
+    color: var(--link-color);
+}
+
+.resources-container {
+    margin-top: 15px;
+    padding: none;
+}
+
+.resources-list {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-template-rows: auto auto 1fr;
+    gap: 1rem;
+}
+
+.resource-link {
+    display: flex;
+    align-items: center;
+    padding: 12px 15px;
+    background: #f5f5f5;
+    border-radius: 8px;
+    text-decoration: none;
+    color: #333;
+    transition: all 0.2s;
+}
+
+.resource-link:hover {
+    background: #e9e9e9;
+    transform: translateY(-2px);
+}
+
+.resource-link:last-child {
+    grid-column: 1 / -1;
+    grid-row: 3;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background-color: #f0f0f0;
+}
+
+.resource-link:last-child .resource-icon {
+    position: relative;
+    top: 4px;
+}
+
+.resource-icon {
+    margin-right: 12px;
+    font-size: 20px;
+}
+
+.resource-link:not(:last-child) .resource-icon {
+    position: relative;
+    top: 2px;
+}
+
+.footer-container {
+    display: grid;
+    margin-top: -30px;
+    margin-right: 24px;
+}
+
+.navigation-buttons {
+    display: flex;
+    justify-content: center;
+    padding: 10px 0 15px 0;
+    gap: 15px;
+}
+
+.nav-button {
```

Comment:
Class names like `header-container`, `footer-container`, `navigation-buttons`, and `nav-button` are too generic to appear at the top-level of the CSS file for the welcome overlay. Two standard fixes are to either

- manually prefix them with something like `welcome-`, or
- turn this file into a CSS module to automatically mangle the names